daemon/swap/SwapOut.go

```diff
@@ -5,19 +5,20 @@ import (
 )
 
 type SwapOut struct {
-	ID                  uint      `gorm:"primaryKey;autoIncrement"`
-	Datetime            time.Time `gorm:"autoCreateTime"`
-	Status              string    `gorm:"type:status_enum;not null" json:"status"`
-	AmountSATS          uint64    `gorm:"not null" json:"amountSats"`
-	DestinationAddress  string    `gorm:"size:255;not null" json:"destinationAddress"`
-	ServiceFeeSatoshis  uint64    `gorm:"not null" json:"serviceFeeSatoshis"`
-	OnchainFeeSatoshis  uint64    `gorm:"not null" json:"onchainFeeSatoshis"`
-	OffchainFeeSatoshis uint64    `gorm:"not null" json:"offchainFeeSatoshis"`
-	DestinationChain    string    `gorm:"type:chain_enum;size:255;not null" json:"destinationChain"`
-	ClaimPubkey         string    `gorm:"size:255;not null" json:"claimPubkey"`
-	PaymentRequest      string    `gorm:"size:255;not null" json:"paymentRequest"`
-	Description         *string   `gorm:"size:255" json:"description"`
-	MaxRoutingFeeRatio  float64   `gorm:"not null" json:"maxRoutingFeeRatio"`
+	ID                 uint       `gorm:"primaryKey;autoIncrement"`
+	Status             SwapStatus `gorm:"type:status_enum;not null"`
+	AmountSATS         uint64     `gorm:"not null"`
+	DestinationAddress string     `gorm:"size:255;not null"`
+	ServiceFeeSATS     uint64     `gorm:"not null"`
+	OnchainFeeSATS     uint64     `gorm:"not null"`
+	OffchainFeeSATS    uint64     `gorm:"not null"`
+	DestinationChain   Chain      `gorm:"type:chain_enum;size:255;not null"`
+	ClaimPubkey        string     `gorm:"size:255;not null"`
+	PaymentRequest     string     `gorm:"size:255;not null"`
```

Comment:
PaymentRequest is still 255 limited, I would remove all size limits for the time being and we can limit it in the future (?)